AEPServices/Sources/utility/hitprocessor/PersistentHitQueue.swift

```diff
@@ -19,13 +19,17 @@ public class PersistentHitQueue: HitQueuing {
     private static let DEFAULT_RETRY_INTERVAL = TimeInterval(30)
     private var suspended = true
     private let queue = DispatchQueue(label: "com.adobe.mobile.persistenthitqueue")
+    private var batchLimit = 0
```

Comment:
My opinion is that the batchLimit should be defined on the `HitProcessing` rather than directly injected into the persistent hit queue. This is so that other uses of HitProcessing can leverage the batch limit, and allows each HitProcessing to decide if they want a setter on their batch limit or an immutable batch limit.

Reply:
Yes that makes sense.